pyproject.toml

```diff
@@ -1,6 +1,40 @@
 [build-system]
-requires = ["setuptools>=45", "wheel", "setuptools_scm>=6.2"]
+requires = ["setuptools>=69", "wheel", "setuptools_scm>=8"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "canopen"
+authors = [
+    {name = "Christian Sandberg", email = "christiansandberg@me.com"},
```

Comment:
Feel free to add more people here. 🙂

Reply:
Alright, I added myself and @af-silva, completing the top-three list of contributors according to GitHub.

Email address taken from one of your commits, @af-silva, hope that's alright?

Reply:
@acolomb It is ok :)

Reply:
I counted up the PRs that go into this release, and a large fraction of them are mine. Please feel free to use my name and email (sveinse@seldal.com) in any release notes.